pkgs/misc/vim-plugins/update.py

```diff
@@ -386,6 +416,31 @@ def generate_nix(plugins: List[Tuple[str, str, Plugin]], outfile: str):
     print(f"updated {outfile}")
 
 
+def update_redirects(input_file: Path, output_file: Path, redirects: dict):
+    with fileinput.input(input_file, inplace=True) as f:
+        for line in f:
+            print(redirects.get(line, line), end="")
+    print(
+        f"""\
+Redirects have been detected and {input_file} has been updated. Please take the
+following steps:
+    1. Go ahead and commit just the updated expressions as you intended to do:
+            git add {output_file}
+            git commit -m "vimPlugins: Update"
+    2. If any of the plugin names were changed, add the old names as aliases in
+    aliases.nix
+    3. Make sure the updated {input_file} is still correctly sorted:
+            sort -udf ./vim-plugin-names > sorted && mv sorted vim-plugin-names
```

Comment:
This step would also be unnecessary if we'd just completely rewrite the file.

Reply:
I agree it would be good if `update.py` automatically sorted `vim-plugin-names`, though that might be out of scope for this PR.

Reply:
Wouldn't that actually reduce complexity?

Reply:
I agree that it would be nice, but I think it's out of scope of the PR, and can be done later.

Reply:
Now that the `--update-rewrite` flag is gone and we're modifying the input file without user input we might as well sort it while we're at it. As @timokau points out it is basically free.